src/ValidatedNumerics.jl

```diff
@@ -55,6 +57,12 @@ end
 export
     IntervalBox, @intervalbox
 
+## Decorations
+export
+    @decorated,
+    interval, decoration, DecoratedInterval,
```

Comment:
I'll change it to `interval_part`. I think it is ok to simply export it; it is actually used in the tests.